src/core/eme/utils/open_sessions_store.ts

```diff
@@ -173,9 +174,11 @@ export default class MediaKeySessionsStore {
   public closeAllSessions() : Observable<null> {
     return observableDefer(() => {
       const disposed = this._entries.map((e) => this.closeSession(e.session));
-      this._entries = [];
       return observableConcat(
-        observableMerge(...disposed).pipe(ignoreElements()),
+        observableMerge(...disposed).pipe(
+          tap(() => this._entries = []),
```

Comment:
This reset entries each time a session is closed.
Either we should use ``finalize``, or we should map to remove the corresponding entry each time